src/Magneto/Core/CacheEntry.cs

```diff
@@ -4,4 +4,12 @@ public class CacheEntry<T>
 	{
 		public T Value { get; set; }
 	}
+
+	public static class CacheEntryExtensions

```

Comment:
Pro tip (from Martin C), add `[EditorBrowsable(EditorBrowsableState.Never)]` to classes in libraries that contain only extension methods. Keeps the browsable apis down in intellisense.

Reply:
Thanks. Done!